.github/scripts/pr_summarizer.py

```
@@ -0,0 +1,63 @@
+import requests
+
+# --- Config ---
+mistral_api_key = os.getenv("MISTRAL_API_KEY")
+github_token = os.getenv("GITHUB_TOKEN")
+repo = os.getenv("GITHUB_REPOSITORY")
+pr_number = os.getenv("PR_NUMBER")
+
+# --- Récupérer le diff de la PR ---
+files_url = f"https://api.github.com/repos/{repo}/pulls/{pr_number}/files"
+headers = {"Authorization": f"token {github_token}"}
+files = requests.get(files_url, headers=headers).json()
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)